templates/CRM/Admin/Form/ScheduleReminders.tpl

```diff
@@ -144,11 +144,12 @@
             </tr>
           </table>
             {include file="CRM/Contact/Form/Task/EmailCommon.tpl" upload=1 noAttach=1}
-        </div>
-      </fieldset>
+        </div><!-- /.crm-accordion-body -->
```

Comment:
`<nitpick>`
  I don't mean to be a pain... but I really don't like these comments. It's 2024 and we all know what a closing tag is.
`</nitpick> <!-- end of nitpick -->`

Reply:
Yes, especially as it sits next to the closing details tag (or should be). Perhaps at the end of this there can be a civi wide cleanup of `<!-- /.crm-accordion-wrapper -->` & `<!-- /.crm-accordion-body -->?

Reply:
Sure. Ideally this PR wouldn't be adding new ones that we then have to cleanup.
But if that's easier then ok I can merge this.

Reply:
Those comments are now removed for all the templates in #29448